moped-database/migrations/1682011649641_create_new_subcomponents/up.sql

```diff
@@ -0,0 +1,24 @@
+INSERT INTO moped_subcomponents (subcomponent_name) values
+    ('Continuous flow intersection'),
+    ('Curb extension'),
+    ('Detection - Bicycle'),
+    ('Vehicle lane reconfiguration'),
+    ('Protected intersection - Full'),
+    ('Protected intersection - Full - Shared use'),
+    ('Protected intersection - partial'),
+    ('Radius tightening'),
+    ('Slip lane removal'),
+    ('Smart right'),
+    ('Intersection square-up'),
+    ('Transit corner modification'),
+    ('Bicycle accommodations at PHB'),
+    ('Bicycle signal (standard signal + bicycle signal sign)'),
+    ('Bicycle signal face (experiment)'),
+    ('Bicycle signal face (interim approval)'),
+    ('Detection - vehicle'),
```

Comment:
same question here. Detection - Bicycle but here its lower case vehicle

Reply:
I fixed this - thank you